example/app/assets/javascripts/flash_messages.js

```diff
@@ -0,0 +1,17 @@
+var eventName = typeof(Turbolinks) !== 'undefined' ? 'turbolinks:load' : 'DOMContentLoaded';
+
+if (!document.documentElement.hasAttribute("data-turbolinks-preview")) {
+  document.addEventListener(eventName, function flash() {
+    var flash = JSON.parse(document.getElementById('flash').dataset.flash);
+
+    if (flash.notice) {
+      ShopifyApp.flashNotice(flash.notice);
```

Comment:
Not a dealbreaker on merging this PR, but my preference is that (if we’re editing this stuff anyway) we stop using EASDK and start using App Bridge. cc @gf3